tests/plasticity/README.md

```diff
@@ -0,0 +1,39 @@
+This folder contains everything that is needed to compile, run and test a preliminary implementation of the sinexp and alpha synaptic plasticity model.
+
+The subfolders `custom_stdp` contains the source code for needed version of the cerebellum that we will test.
+The first one contains four E-GLIF models for Granule cell (`eglif_cond_alpha_multisyn`), Inferior Olivary cell (`eglif_io_nestml`), Purkinje cell (`eglif_pc_nestml`) and, MLI (`eglif_mli`)
+
+To compile these extension modules, create a build directory at the same level of the folder containing the source code. Inside the `plasticity` folder:
+```
+mkdir build_dir
+```
+Go inside it:
+```
+cd build_dir
+```
+If it's not the first time you compile the module in this build directory, remove all files:
+```
+rm -rf *
+```
+Then compile the module by running:
+```
+cmake -Dwith-nest={path of the your nest} {way of your custom stdp like}  #As an example => cmake -Dwith-nest=${NEST_INSTALL}/bin/nest-config /home/alphabuntu/test2/Tesi/plasticity/custom_stdp
```

Comment:
maybe better `cmake -Dwith-nest={path of your nest-config} {path to custom_stdp folder}`